cat-theme/src/css/base/_buttons.scss

```diff
@@ -0,0 +1,47 @@
+.btn {
+    &--main {
+
+        @include flex ($justify-content : flex-start);
+
+        border: 0.5px solid $color-header-text-light;
+        border-radius: 4rem;
+        height: 5rem;
+        font-size: 1.5rem;
+        transition: all 0.5s ease-in-out;
+        width: 17rem;
+
+        &:hover {
+            transform: scale(1.1) perspective(1px);
+        }
+
+        a {
+
+            @include flex ($justify-content : flex-start);
+
+            cursor: pointer;
+        }
+        
+        &--arrow {
+
+            @include flex;
+            
+            background-color: $color-button-border-light;
+            border-radius: 50%;
+            color: $color-button-text-light;
+            height: 3rem;
+            margin-left: 2rem;
+            width: 3rem;
+        }
+
+        &--text {
+            margin-left: 2rem;
+
+            p {
```

Comment:
Are we really setting `<p />` tags with special CSS or should this be applied to other text as well?